src/v4l2.rs

```diff
@@ -32,18 +43,30 @@ mod ll {
             prot: c_int,
             flags: c_int,
             fd: RawFd,
-            offset: off_t,
+            offset: i64,
         ) -> *mut c_void;
         pub fn v4l2_munmap(start: *mut c_void, length: size_t) -> c_int;
     }
 }
 
 #[cfg(feature = "no_wrapper")]
 mod ll {
-    use libc::{c_int, c_ulong, c_void};
+    use libc::{c_int, c_ulong, c_void, off_t, size_t};
     use std::os::unix::io::RawFd;
 
-    pub use libc::{close, mmap, munmap, open};
+    pub use libc::{close, munmap, open};
+
+    pub fn mmap(
+        start: *mut c_void,
+        length: size_t,
+        prot: c_int,
+        flags: c_int,
+        fd: RawFd,
+        offset: off_t,
+    ) -> *mut c_void {
+        // Wrap away unsafety to match the other mmap wrapper function
```

Comment:
I'm convinced `mmap` should still be `unsafe fn`.

Reply:
Sure, np.